### PR TITLE
Vibe 2.0 - css modules migration 5

### PR DESCRIPTION
**Pull request by SergeyRoyt**
- [ ] a - https://github.com/mondaycom/monday-ui-react-core/pull/846
  - [ ] 1. AlertBanner
  - [ ] 2. AlertBannerButton
  - [ ] 3. AlertBannerLink
  - [ ] 4. AlertBannerText
- [ ] b - https://github.com/mondaycom/monday-ui-react-core/pull/850
  - [ ]  5. Accordion
  - [ ]  6. AttentionBox
  - [ ]  7. Avatar
  - [ ]  8. AvatarBadge
  - [ ]  9. AvatarContent

### CODE REVIEW

File: src/components/Accordion/AccordionItem/AccordionItem.tsx [hunk 1]
```diff
@@ -56,14 +57,15 @@ const AccordionItem: React.FC<AccordionItemProps> = forwardRef(
...
-      <div ref={mergedRef} className={cx("accordion-item", className)} id={id}>
+      <div ref={mergedRef} className={className} id={id}>
         <ExpandCollapse
           iconSize={iconSize}
           id={`expand-collapse--${id}`}
           onClick={onClickCallback}
           open={open}
           title={title}
           hideBorder={hideBorder}
+          componentClassName={expandCollapseComponentClassName}
```
Comment (hadasfa) [inline src/components/Accordion/AccordionItem/AccordionItem.tsx:68]: no classname prpo?

File: src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@import "../../../styles/global-css-settings";
+@import "../../../styles/typography";
+@import "../../../styles/themes";
+
+.bannerButton {
+  // TODO Mismatched property value - fix other
```
Comment (hadasfa) [inline src/components/AlertBanner/AlertBannerButton/AlertBannerButton.module.scss:6]: ?

File: src/components/AlertBanner/__tests__/__snapshots__/AlertBanner-snapshots.jest.js.snap [hunk 3]
```diff
@@ -3,35 +3,38 @@
...
 <div
   aria-label="banner"
-  className="monday-alert-banner monday-alert-banner--background-color-primary"
+  className="alertBanner primary"
+  data-testid="alert-banner"
   role="banner"
 >
   <div
-    className="monday-alert-banner__inner"
+    className="inner"
```
Comment (hadasfa) [inline src/components/AlertBanner/__tests__/__snapshots__/AlertBanner-snapshots.jest.js.snap:11]: inner => alertBannerContent?

Comment (hadasfa) [reply src/components/AlertBanner/__tests__/__snapshots__/AlertBanner-snapshots.jest.js.snap:11]: or content? inner is weird

File: src/components/AttentionBox/AttentionBox.module.scss [hunk 4]
```diff
@@ -0,0 +1,83 @@
+@import "../../styles/themes";
+@import "../../styles/typography";
+@import "../../styles/global-css-settings";
+
+.boxComponent {
```
Comment (hadasfa) [inline src/components/AttentionBox/AttentionBox.module.scss:5]: boxComponent => attentionBox

File: src/components/EditableHeading/EditableHeading.tsx [hunk 5]
```diff
@@ -211,7 +212,7 @@ const EditableHeading: React.FC<EditableHeadingProps> & {
       })}
       aria-label={`${value} ${tooltip || ""}`}
       id={id}
-      data-testid={dataTestId || dataTestIdOverride}
```
Comment (hadasfa) [inline src/components/EditableHeading/EditableHeading.tsx:214]: dataTestIdOverride become default so should be taken if we have both values

File: src/components/Accordion/AccordionItem/AccordionItem.tsx [hunk 6]
```diff
@@ -56,14 +57,15 @@ const AccordionItem: React.FC<AccordionItemProps> = forwardRef(
     }, [onClickAccordionCallback, onClick]);
 
     return (
-      <div ref={mergedRef} className={cx("accordion-item", className)} id={id}>
+      <div ref={mergedRef} className={className} id={id}>
```
[the remark this answers is not in the document]
Comment (SergeyRoyt) [reply src/components/Accordion/AccordionItem/AccordionItem.tsx:60]: exactly

File: src/components/Accordion/AccordionItem/AccordionItem.tsx (hunk 1 above)
Comment (SergeyRoyt) [reply src/components/Accordion/AccordionItem/AccordionItem.tsx:68]: All good, `expand-collapse` was the one which was getting overriding 
image

**Pull request closed without merging**